.environment/gitleaks/gitleaks-config.toml

```diff
@@ -54,6 +55,7 @@ title = "PRIME ReportStream Gitleaks Configuration"
             '(?i)cbrown@cdc.gov',
             '(?i)cglodosky@cdc.gov',
             '(?i)data@cdc.gov',
+            '(?i)default@noemail.com',
```

Comment:
I had to add this after I merged in master. I'm not sure how this got into master but broke my gitleaks check.